src/components/light.js

```diff
@@ -46,10 +46,37 @@ module.exports.Component = registerComponent('light', {
       // Light type has not changed. Update light.
       Object.keys(diffData).forEach(function (key) {
         var value = data[key];
-        if (['color', 'groundColor'].indexOf(key) !== -1) {
-          value = new THREE.Color(value);
+
+        switch (key) {
+          case 'color':
+            light[key] = new THREE.Color(value);
```

Comment:
Since i'm checking for specific keys in the switch, i could also set the light.angle and light.target values directly, instead of with light[key] = ... would i change that too ?